ArgoServer/src/server/SendGetMethods.cpp

```diff
@@ -0,0 +1,199 @@
+﻿#include "stdafx.h"
+#include "server/Server.h"
+
```

Comment:
Entire contents of file should be in `Server.cpp`
Change all occurrences of `char*` to `std::byte*`
all casts of `(char*)` need to be changed to `reinterpret_cast<std::byte*>()`